aces/Loris
# [API] Added features for dicoms : PatientName and images: isPhantom

## Brief summary of changes

`Patientname` is added to dicoms endpoint view and `IsPhantom` is added to images endpoint view

#### Testing instructions

1. Go to `<hostname>/api/v0.0.3/candidates/400162/V6/dicoms/`. Under the attribute `Tarname`, a new attribute should be there: `Patientname : "ROM162_400162_V6"`


2. Go to `<hostname>/api/v0.0.3/candidates/400162/V6/images/`. Each file should have 4 attributes, the last one being the new attribute, `IsPhantom: false`
